app/scripts/modules/core/src/pipeline/config/triggers/artifacts/helm/HelmArtifactEditor.tsx

```diff
@@ -54,7 +54,7 @@ class HelmEditor extends React.Component<IArtifactEditorProps, IHelmArtifactEdit
             value={artifact.name || ''}
             onChange={(e: Option) => {
               this.onChange(e, 'name');
-              this.onNameChange();
+              this.onNameChange(e.value.toString());
```

Comment:
Is the value not already a string?

Reply:
Probably? I was just being consistent with other places in the file.